packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -205,10 +156,22 @@ class _TimePickerHeader extends StatelessWidget {
     onChanged(selectedTime.replacing(hour: newHour));
   }
 
+  TextStyle _getBaseHeaderStyle(TextTheme headerTextTheme) {
+    assert(orientation != null);
+    switch (orientation) {
+      case Orientation.portrait:
+        return headerTextTheme.display3.copyWith(fontSize: 60.0);
```

Comment:
The spec doesn't say what sizes to use.  If we use these sizes, we get pixel identical results to the screenshots in the spec.  /me will document.